src/odemis/util/filename.py

```diff
@@ -103,29 +103,29 @@ def guess_pattern(fn):
                 fn_ptn = fn_ptn[0:m.start()] + name_ptn + fn_ptn[m.end():]
 
     # Detect count
-    cnt_ptn = r'\d{1,5}'
-    cnt_m = None
-    for m in re.finditer(cnt_ptn, fn_ptn):
-        cnt_m = m
-    # if multiple numbers are present, use last
-    if cnt_m:
-        cnt = cnt_m.group()
-        fn_ptn = fn_ptn[:cnt_m.start()] + "{cnt}" + fn_ptn[cnt_m.end():]
-    else:
-        cnt = "001"  # will be used in case cnt pattern is added afterwards
-
-    # If neither time, nor count are specified, add count, if only short time (h, min)
-    # specified and no count, add seconds to make filename unique
-    # Doesn't behave properly if user enters terms with curly braces like {{cnt}}.
-    if '{cnt}' not in fn_ptn:
-        if '{timeshrt_colon}' in fn_ptn:
-            fn_ptn = re.sub('{timeshrt_colon}', '{timelng_colon}', fn_ptn)
-        elif ("{timelng}" in fn_ptn or
-              "{timelng_colon}" in fn_ptn or
-              "{timelng_hyphen}" in fn_ptn):
-            pass
-        else:
-            fn_ptn = fn_ptn + '-{cnt}'
+    cnt = "001"  # will be used in case cnt pattern is added afterwards
+    if is_count:
```

Comment:
Please add (at least) one test case, which calls the function with `is_count=False`.